lib/davinci_dtr_test_kit/client_groups/adaptive_questionnaire/dtr_adaptive_next_question_request_test.rb

```diff
@@ -66,6 +66,11 @@ def prompt_cont
     end
 
     run do
+      if respond_to?(:custom_next_question_questionnaire)
```

Comment:
is this still needed with the update to separate out the adaptive dinner tests?

Reply:
nope updated